locales/afr/LC_MESSAGES/messages.po

```diff
@@ -5,34 +5,521 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: \n"
-"POT-Creation-Date: 2021-07-19 16:02+0200\n"
-"PO-Revision-Date: 2021-07-20 15:06+0200\n"
+"POT-Creation-Date: 2021-07-20 15:37+0200\n"
+"PO-Revision-Date: 2021-07-20 17:16+0200\n"
+"Last-Translator: \n"
 "Language-Team: \n"
+"Language: af\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: pygettext.py 1.5\n"
 "X-Generator: Poedit 3.0\n"
-"Last-Translator: \n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"Language: af\n"
 
-#: vaccine/vaccine_reg_whatsapp.py:82
+#: vaccine/healthcheck_vacreg_ussd.py:41
+msgid "Welcome to the NATIONAL DEPARTMENT OF HEALTH's COVID-19 Response service"
+msgstr "HOOFKIESLYS [0]"
+
+#: vaccine/healthcheck_vacreg_ussd.py:45
+msgid "ERROR: Please try again"
+msgstr "FOUT: Probeer asseblief weer"
+
+#: vaccine/healthcheck_vacreg_ussd.py:47
+msgid "Vaccine Registration"
+msgstr "EVDS-Registrasie vir Inenting"
+
+#: vaccine/healthcheck_vacreg_ussd.py:50
+msgid "HealthCheck Symptom checker [ENG]"
+msgstr "HealthCheck Symptom checker [ENG]"
+
+#: vaccine/healthcheck_vacreg_ussd.py:52
+msgid "Change language"
+msgstr "Verander taal"
+
+#: vaccine/healthcheck_vacreg_ussd.py:89 vaccine/healthcheck_vacreg_ussd.py:93
+msgid ""
+"Welcome back to the NATIONAL DEPARTMENT OF HEALTH's COVID-19 Vaccine "
+"Registration service"
+msgstr ""
+"Welkom terug by die NASIONALE DEPARTMENT VAN GESONDHEID se COVID-19-inentings-"
+"registrasiediens"
+
+#: vaccine/healthcheck_vacreg_ussd.py:100 vaccine/healthcheck_vacreg_ussd.py:122
+msgid "Continue where I left off"
+msgstr "Gaan voort waar ek opgehou het"
+
+#: vaccine/healthcheck_vacreg_ussd.py:102 vaccine/healthcheck_vacreg_ussd.py:124
+msgid "Start over"
+msgstr "Begin oor"
+
+#: vaccine/healthcheck_vacreg_ussd.py:109 vaccine/healthcheck_vacreg_ussd.py:114
+msgid ""
+"Welcome back to The National Department of Health's COVID-19 Service\n"
+"\n"
+"Reply"
+msgstr ""
+"Welcome back to The National Department of Health's COVID-19 Service\n"
```

Comment:
Should this not be the "Welkom terug.." translation?

Reply:
No, this is the welcome back for the healthcheck, the `Welcome back to the NATIONAL DEPARTMENT OF HEALTH's COVID-19 Vaccine Registration service` is the session resume for registration

Reply:
👍